src/components/main/Header/index.jsx

```diff
@@ -1,17 +1,33 @@
 import {BiMenuAltLeft, BiUser} from "react-icons/bi";
+import {useRef, useState} from "react";
 import Logo from "components/main/Logo";
 import HeaderBtn from "components/main/Header/components/HeaderBtn";
+import DropDown from "routes/Dashboard/DropDown";
 
 export default function Header(){
+    const dropDownRef = useRef(null);
+    const [showProfile, setShowProfile] = useState(false)
+    const handleMenu = () => {
+        console.log('Menu')
+    }
+
+    const handleProfile = () => {
+        if(!showProfile)
+            setShowProfile(true)
+        else
+            setShowProfile(false)
+    }
+
     return (
-        <header className="w-full flex justify-between items-center bg-white">
-            <HeaderBtn>
+        <header ref={dropDownRef} className="relative w-full flex justify-between items-center bg-white">
+            <HeaderBtn onClick={handleMenu}>
```

Comment:
get rid of this `onClick` until you're really going to implement it in future PRs